### DISCUSSION TITLE
problem...

### DISCUSSION THREAD

**Discussion by Dusty1345** [FunkinCrew/Funkin#1704] [Compiling help] [answered]
i have a problem, what i need to do?
![Captura de pantalla (5)](https://user-images.githubusercontent.com/88300541/131237399-5ae5691d-5ca4-4e22-bb3b-ed0e29a0b305.png)

**Comment by libauror-a** [comment, marked as answer]
add apistuff and copy the stuff from the readme

**Reply by Dusty1345** [reply]
OMG THX, IT WORKED :')

**Reply by libauror-a** [reply]
yea, ninjamuffin will probally find a better solution to the apistuff stuff.
